### DISCUSSION TITLE
WIP: Mod Conversion and Optimization Pass

### DISCUSSION THREAD

**Discussion by jschmidt92** [NikolaiF90/ArmaPersistentCoreSystem#6] [General]
I've started working on converting this framework to a mod, implementing optimizations and refactoring directory structure.

**Comment by NikolaiF90** [comment]
I'm not really sure if this system should have the mod version. There's a few things about cinverting it to mod. One of it is target audience. If we're aiming for mission makers, i dont think thats a problem. But for player user, that'll be a problem if they tried to load it into other people's mission. But Im excited to see what you could come up with. And I really appreciate the work you put in

**Reply by jschmidt92** [reply]
Understandable which is why I'm designing it to be either be used player side or server side, depending on how the mission is setup, and I'll have an example mission template and docs to go with it. You can add me on discord and I can show you what I've done so far if you would like.

**Reply by NikolaiF90** [reply]
Sure. What's your discord
